### PR TITLE
feat: widget v3 alpha

### PR DESCRIPTION
**Pull request by Addminus**
This PR implements the v3 aplha version of the widget. It encompasses
- allows for connecting solana wallets to jumper
- introduces a new ecosystem selection menu. A use may now select whether he connects to the evm or svm connection of a wallet that offers both ecosystems
- introduces a new wallet menu in order to connect and manage one evm and one svm wallet at the same time
- a lot of bumped packages
- New hooks to manage accounts and wallets

This PR also changes some underlying systems:
- User tracking has to be adjusted to accommodate the new ecosystems
- Menu state management has be refactored: references for anchor elements are not passed via zustand stores anymore, changed naming system of state management hooks from "on.....MenuOpened" to "set....MenuState"
- Menus are not registered globally in the `App` component anymore.
- Mobile Menu has been adjusted to use the MUI native `Drawer` component instead of a custom `Popper` based system.

### CODE REVIEW

File: src/components/Menus/EcosystemSelectMenu/EVMConnectButton.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+export const EVMConnectButton = ({ walletIcon, evm }: EvmConnectButton) => {
+  const theme = useTheme();
+  const connect = useAccountConnect();
+
+  const { closeAllMenus } = useMenuStore((state) => state);
+
+  const connectHandler = async () => {
+    const identityCheckPassed = await isWalletInstalledAsync(evm.id);
+    if (!identityCheckPassed) {
+      //   onNotInstalled(connector);
```
Comment (DNR500) [inline src/components/Menus/EcosystemSelectMenu/EVMConnectButton.tsx:22]: Is this still needed? If you want to keep this in might be good to add a comment to say why

File: src/components/Widgets/WidgetEvents.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { useUserTracking } from 'src/hooks';
 
 import type {
   ChainTokenSelected,
-  RouteContactSupport,
+  // RouteContactSupport,
```
Comment (DNR500) [inline src/components/Widgets/WidgetEvents.tsx:6]: Do we need to keep this?

File: src/components/Widgets/WidgetEvents.tsx [hunk 3]
```diff
@@ -191,7 +191,7 @@ export function WidgetEvents() {
     );
     widgetEvents.on(WidgetEvent.RouteExecutionFailed, onRouteExecutionFailed);
     widgetEvents.on(WidgetEvent.RouteHighValueLoss, onRouteHighValueLoss);
-    widgetEvents.on(WidgetEvent.RouteContactSupport, onRouteContactSupport);
+    // widgetEvents.on(WidgetEvent.RouteContactSupport, onRouteContactSupport);
```
Comment (DNR500) [inline src/components/Widgets/WidgetEvents.tsx:194]: Again maybe a line to help understand why we are keeping this in?

**Review by chybisov — CHANGES_REQUESTED** [review]
image


When on a dark theme those borders should have adjusted color to have a cut effect as on a light theme. Also, they look too thick and we should make them a little thinner on both themes.

**Comment by chybisov** [comment]
@Addminus the above comment can be related to https://github.com/lifinance/jumper.exchange/pull/749

**Comment by dennyscode** [comment]
@Addminus @chybisov Tweaked just little things to follow along with styling.

Few things to bring up:

1.) Let´s talk again about these wallet logos and check MetaMask which seems to be pretty squeezed into that circle. On Taho it looks good. But please confirm once again, Eugene: We will handle such cases like the MetaMask after we will update logos to make them all fit well, right?

2.) The new wallet-menu looks nice! Good job. But now, seeing this one and clicking through: It feels odd being at the "Select Wallet" menu.
Would it be better to use same typography for "Connect another wallet" and "See all wallets" buttons?
image
image

**Comment by chybisov** [comment]
@dennyscode 

1) Yes, we will update them so they fit nicely.
2) Yes, let's make typography consistent.

**Review by chybisov — CHANGES_REQUESTED** [review]
@Addminus found one more issue with wallet menu - when you are disconnected it is not possible to see all wallets.

**Comment by chybisov** [comment]
@Addminus @dennyscode I fixed and released some updates for wallet icons, so it should be better now, especially for MetaMask. We will continue to update more icons soon.

**Comment by Addminus** [comment]
> @Addminus found one more issue with wallet menu - when you are disconnected it is not possible to see all wallets.

😂 I have a déjà-vue

**Comment by chybisov** [comment]
@Addminus, we were too optimistic to think that everyone followed the standards 😄 Solflare doesn't automatically inject itself and many others too. I added Solflare manually, let's figure out what else we need to add 🙂 Here is the list https://github.com/anza-xyz/wallet-adapter/tree/master/packages/wallets

So far, I found Phantom and Backpack follow standards.

**Comment by dennyscode** [comment]
One more bug to be fixed: When connected to SOL, the avatar seems to already come with  chain badge:
image

**Comment by chybisov** [comment]
@dennyscode It is not particularly a bug, more like a shitty icon from Solflare MM Snap 😄 I guess we will need to come up with a workaround for this 🥲

**Comment by Addminus** [comment]
> @Addminus found one more issue with wallet menu - when you are disconnected it is not possible to see all wallets.

Fixed in last commit

**Pull request merged**